charter/index.md

```diff
@@ -0,0 +1,228 @@
+
+Deliverables and supporting documents
+produced by this Community Group are
+hosted on the group’s [GitHub](https://github.com/w3c/font-text-cg).
+
+The group uses the private internal-font-text@w3.org mailing list ([archives](https://lists.w3.org/Archives/Member/internal-font-text/))
+for administrative purposes,
+such as organizing meetings,
+requesting commit access to GitHub,
+etc.
+
+Additionally, the group will use
+the #font-text channel on the [W3C IRC server](https://www.w3.org/Project/IRC/)
+during meetings
+and for day-to-day casual chatting.
+
+## Decision Policy
+
+This group will seek to make decisions when there is consensus,
+and favors proposals that create the weakest objections.
+This is preferred over proposals that are supported by a large majority
+but that cause strong objections from a few people.
+
+In some cases,
+even after careful consideration of all points of view,
+the group might find itself unable to reach consensus.
+The Chair(s) may record a decision where there is dissent,
+while minuting objections,
+so that the group can make progress.
+Dissenters cannot stop a group’s work
+simply by saying that they cannot live with a decision.
+When the Chair(s) believe that the Group has
+duly considered the legitimate concerns of dissenters
+as far as is possible and reasonable,
+the group should move on.
+
+In addition to decisions made
+on teleconferences or face to face meetings,
+decisions may also be made by a call for consensus
+on the public mailing list;
+consensus to be determined by the Chair(s)
+after some reasonable interval for objections.
+
+All decisions made by the group should be considered resolved
+unless and until new information becomes available
+or unless reopened at the discretion of the Chair(s).
+
+The Chair(s) may appoint individuals to be the
+Editors of the Community Group's Deliverables.
+Such Editors are bound by the consensus of the Community Group
+as determined by the Chair(s),
+and being an Editor does not imply any independent authority
+over the content of the deliverable.
+Editors may nonetheless be allowed by the Chair(s)
+to make non controversial edits
+even in the absence of an explicit group decision,
+or in order to propose some text to be reviewed by the Community Group.
+Even then, if questions or disagreements arise,
+the final determination of consensus remains with the Chair(s).
+
+## Contribution Mechanics
+
+Community Group participants agree to make contributions
+in the GitHub repo
+for the project that they are interested in.
+This may be done by submitting a pull request,
+by raising an issue,
+or by adding a comment to an existing issue.
+The Community Group mailing list must not be used
+for discussing details of specific projects.
+
+Documents maintained by the Community Group
+must use the W3C Software and Document License.
+
+All GitHub repositories attached to the Community Group
+must contain a copy of the CONTRIBUTING and LICENSE files.
```

Comment:
Link to https://github.com/w3c/font-text-cg/blob/main/LICENSE.md. And note we don’t even have a “CONTRIBUTING” file yet…

Reply:
Sure, but it makes sense that we will and I think it's better to only have to link to the charter later, not add wording that could be viewed and a substance change.